Cargo.toml

```diff
@@ -65,10 +65,14 @@ serde = "1.0.159"
 serde_yaml = "0.9.19"
 
 [features]
-default = ["electrum", "log"]
 all = ["cli", "log"]
+default = ["cli", "electrum", "log"]
```

Comment:
Why do we need `cli` feature as a default? Libraries do not need it

Reply:
Probably fine to remove it from here